src/nodes/core/NodeFunctionInput.js

```diff
@@ -1,11 +1,55 @@
+/**
+ * Describes the input of a {@link NodeFunction}.
+ */
 class NodeFunctionInput {
 
+	/**
+	 * Constructs a new node function input.
+	 *
+	 * @param {String} type - The input type.
+	 * @param {String} name - The input name.
+	 * @param {Number?} [count=null] - TODO (only relevant for GLSL).
```

Comment:
@sunag Not sure about the purpose of `count`.

It also seems this part of the code throws currently a runtime error in `GLSLNodeFunction`. Try [webgpu_materials](https://threejs.org/examples/webgpu_materials) with a WebGL backend.

> Uncaught TypeError: undefined has no properties
    parse$1 https://threejs.org/build/three.webgpu.js:27563
    GLSLNodeFunction https://threejs.org/build/three.webgpu.js:27607
    parseFunction https://threejs.org/build/three.webgpu.js:27655